### PR TITLE
Alerting: Adds evaluation interval to group view

### PR DESCRIPTION
**Pull request by gillesdemey**
This change adds the evaluation interval to the summary view for evaluation groups.

Also adds some more colors to the rule summary and always shows recording and normal alert rules in the stats.

Screenshot 2022-12-07 at 17 58 49

### CODE REVIEW

**Review by soniaAguilarPeiron — APPROVED** [review]
LGTM! 🙌
I added a non-blocking question about removing total count.

File: public/app/features/alerting/unified/components/rules/RuleStats.tsx [hunk 1]
```diff
@@ -41,75 +42,65 @@ export const RuleStats: FC<Props> = ({ showInactive, showRecording, group, names
...
     );
   }
-  if (showInactive && calculated[PromAlertingRuleState.Inactive]) {
+
+  if (calculated[PromAlertingRuleState.Inactive]) {
     statsComponents.push(
-      <StateColoredText key="inactive" status="neutral">
-        {calculated[PromAlertingRuleState.Inactive]} normal
-      </StateColoredText>
+      <Badge color="green" key="inactive" text={`${calculated[PromAlertingRuleState.Inactive]} normal`} />
```
Comment (konrad147) [inline public/app/features/alerting/unified/components/rules/RuleStats.tsx:77]: I wonder if we should display a badge for `Inactive/Normal` state. It doesn't exist for Mimir/Prom rules so the behaviour will be a bit inconsistent here

**Review by konrad147 — APPROVED** [review]
Looks great! One minor question

File: public/app/features/alerting/unified/components/rules/RuleStats.tsx (hunk 1 above)
Comment (armandgrillet) [reply public/app/features/alerting/unified/components/rules/RuleStats.tsx:77]: I believe ease of use is more important than consistency as long as we keep track of what we could land upstream to eventually reach consistency.

File: public/app/features/alerting/unified/components/rules/RuleStats.tsx [hunk 2]
```diff
@@ -41,75 +42,65 @@ export const RuleStats: FC<Props> = ({ showInactive, showRecording, group, names
...
 
+  const hasStats = Boolean(statsComponents.length);
+
   return (
-    <div>
-      <span>
-        {calculated.total} {pluralize('rule', calculated.total)}
-      </span>
-      {!!statsComponents.length && (
+    <Stack direction="row">
```
[the remark this answers is not in the document]
Comment (gillesdemey) [reply public/app/features/alerting/unified/components/rules/RuleStats.tsx:88]: Yea I have it removed for the evaluation groups but should maybe bring it back for the summary on the top of the page?

**Pull request merged**